prod-devel-rcar.yaml

```diff
@@ -169,7 +169,7 @@ components:
         - [INIT_MANAGER, "systemd"]
 
         # Do not install kernel image to rootfs to decrease initrd size
-        - ["RDEPENDS:${KERNEL_PACKAGE_NAME}-base", ""]
+        - ["RRECOMMENDS:${KERNEL_PACKAGE_NAME}-base", ""]
```

Comment:
Why old way is not working anymore? Commit message does not clarify this.

Reply:
Hi @lorc,

I've partially mentioned the reasoning in my original PR:
https://github.com/otyshchenko1/meta-xt-prod-devel-rcar/pull/1

```
Used yocto FAQ:
https://docs.yoctoproject.org/4.0.10/kernel-dev/faq.html
Paragraph 6.1.4 - How do I install/not-install the kernel image on the root filesystem?
```

That paragraph contains such a statement:

```
The kernel image (e.g. vmlinuz) is provided by the kernel-image package. Image recipes depend on kernel-base. To specify whether or not the kernel image is installed in the generated root filesystem, override **RRECOMMENDS:**${KERNEL_PACKAGE_NAME}-base to include or not include “kernel-image”. See the “[Appending Other Layers Metadata With Your Layer](https://docs.yoctoproject.org/dev/dev-manual/layers.html#appending-other-layers-metadata-with-your-layer)” section in the Yocto Project Development Tasks Manual for information on how to use an append file to override metadata.
```

So, I've followed the recent Yocto documentation.

After your comment, I've compared it with the Dunfell FAQ:

```
https://docs.yoctoproject.org/3.1.25/kernel-dev/kernel-dev-faq.html
```

And it has the following statement:

```
The kernel image (e.g. vmlinuz) is provided by the kernel-image package. Image recipes depend on kernel-base. To specify whether or not the kernel image is installed in the generated root filesystem, override **RDEPENDS_**${KERNEL_PACKAGE_NAME}-base to include or not include “kernel-image”. See the “[Using .bbappend Files in Your Layer](https://docs.yoctoproject.org/3.1.25/dev-manual/dev-manual-common-tasks.html#using-bbappend-files-in-your-layer)” section in the Yocto Project Development Tasks Manual for information on how to use an append file to override metadata.
```

So, I was simply following the change documentation. I guess we might extend the commit message with a more detailed explanation. Would that be sufficient from your point of view?

Reply:
Ah, okay, thanks for the investigation.